stripe/src/main/java/com/stripe/android/view/CountryTextInputLayout.kt

```diff
@@ -167,30 +177,35 @@ internal class CountryTextInputLayout @JvmOverloads constructor(
 
     /**
      * @param countryCode specify a country code to display in the input. The input will display
-     * the full country display name.
+     * the full country display name.  This is preferred over the function that takes a string
```

Comment:
Do we still need the other version?

In any case, I would recommend marking the other method as deprecated instead of adding this comment.